infrastructure_playbook_repo.md

```diff
@@ -0,0 +1,179 @@
+| [hxr.haproxy-error-pages](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.haproxy-error-pages) |  |  | Downloads Galaxy error pages through a bash script _(NOTE: Bash script is not found in the role, so can't explain what these error pages are)_ |
+| [hxr.install-to-venv](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.install-to-venv) |  | :heavy_check_mark: | Installs Python dependencies into any requested virtual environment |
+| [hxr.monitor-cluster](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-cluster) |  | :heavy_check_mark: | Adds Condor cluster monitoring scripts and configures Telegraf user to run these scripts |
+| [hxr.monitor-cvmfs](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-cvmfs) |  | :heavy_check_mark: | Adds a Telegraf task that monitors the CernVM-FS repos |
+| [hxr.monitor-email](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-email) |  | :heavy_check_mark: | Adds an `/var/spool/mail` counter script and adds an Telegraf task to monitor |
+| [hxr.monitor-galaxy](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-galaxy) |  |  | _NOTE: Tasks file is empty_ |
+| [hxr.monitor-galaxy-journalctl](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-galaxy-journalctl) |  | :heavy_check_mark: | Adds a script and a Telegraf task to monitor Galaxy's journalctl logs |
+| [hxr.monitor-galaxy-queue](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-galaxy-queue) |  |  | Adds a Telegraf task to run `gxadmin` queries for monitoring Galaxy queue and workflow-incovcation-status |
+| [hxr.monitor-squid](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-squid) |  | :heavy_check_mark: | Adds a [squid proxy](https://github.com/squid-cache/squid) data parser script and a Telegraf task to monitor |
+| [hxr.monitor-ssl](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.monitor-ssl) |  | :heavy_check_mark: | Adds an SSL check script and a Telegraf task to monitor SSL certificates expiry |
+| [hxr.postgres-connection](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.postgres-connection) |  | :heavy_check_mark: | Adds Galaxy database credentials and required ENVs to the user's bashrc file and creates a `~/.pgpass` file as well |
+| [hxr.remap-user](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/hxr.remap-user) |  | :heavy_check_mark: | Remaps system user's UID and GID to the tomcat user/group |
```

Comment:
This looks obsolete, since we have it in the handy_os role

Reply:
Used by [apollo.yml](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/apollo.yml) playbook.

As it is mentioned in the readme if a role is used in a playbook in the repo then it is marked as _still being used_. Yes, this is obsolete and should be migrated to the one in [os_setup](https://github.com/usegalaxy-eu/ansible-collection-handy/tree/main/roles/os_setup) role in our [handy](https://github.com/usegalaxy-eu/ansible-collection-handy) collection. 

This document describes the current state of the repo. 

We should do cleanup tasks at some point (this document will help us to have an overlook):
1. Remove roles that are not used 
2. Change obsolete roles with the ones that we maintain (like this role for example)